### PR TITLE
docs(docs/repo): add labels list to docs. 

### PR DESCRIPTION
**Pull request by P1sar**
## Changes
Added docs/repo/labels.md documentation that contains a list of labels that will be used in the gossamer repository.
Please confirm the list and structure. After confirmation, I will rename the existing one to the new format (so we do not lose existing marks) and then will add a CI tool that will allow us to automatically edit the list of labels.

## Tests

```sh
go test -tags integration github.com/ChainSafe/gossamer
```

## Primary Reviewer

@timwu20

### CODE REVIEW

**Review by EclesioMeloJunior — APPROVED** [review]

File: docs/docs/repo/labels.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- **`S-`** Scope this work related to, could be multiple, but usually this means that task needs to be break down. **Colours: #1D76DB**
+  - **S-sync-[westend | kusama | polkadot | paseo]** -  related to particular network syncing.
+  - **S-tests** - issue related to adding new tests.  
+  - **S-doc** - documentation related.
+  - **S-cli** - issue related to Gossamer CLI.
+  - **S-ci** - issue related to continuous integration tasks or piplelines.
+  - **S-crypto** - issues related to the lib/crypto package.
+  - **S-grandpa** - issues related to block finality.
+  - **S-babe** - issues related to block production functionality. 
+  - **S-runtime**- issue related to continuous integration tasks or piplelines.
```
Comment (kanishkatn) [inline docs/docs/repo/labels.md:38]: I don't understand the runtime description.

Comment (dimartiro) [reply docs/docs/repo/labels.md:38]: I think `S-ci_cd` could be a best option

**Review by jimjbrettj — APPROVED** [review]
Agree with K's comment on S-runtime, but besides that lgtm

File: docs/docs/repo/labels.md (hunk 1 above)
Comment (P1sar) [reply docs/docs/repo/labels.md:38]: yeah, I basically misstype or rather say miss copy paste

**Review by timwu20 — APPROVED** [review]

File: docs/docs/repo/labels.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+## Labels
+
+Below is the list of labels and their descriptions used in Gossamer repository. 
+
+- **Epic** - Issue used to track development status of a complex feature, aggregates several issues. **Colour: #3E4B9E**
+- **Feature-branch** - pull request from feature branch to origin. **Colour: #8ceac8**
```
Comment (q9f) [inline docs/docs/repo/labels.md:6]: Doesn't that apply to all PRs?

File: docs/docs/repo/labels.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  - **A-tooBig** - issue or PR needs to be broken down to smaller parts.
+  - **A-stale** - issue or PR is deprecated and needs to be closed. 
+  - **A-blocked** - issue or PR is blocked  until something else changes.
+  - **A-triage** - issue desrciption needs refactor and/or labeled. 
+  - **A-debug** - issue requires detective debug work to figure out what's going wrong.
+  - **A-design** - issue requires design work to think about how it would best be accomplished.
+- **`T-`** Describes the type of issue or pull request. **Colour: #FEF2C0**
+  - **T-bug** - this issue covers unexpected and/or wrong behaviour. 
+  - **T-feat** - this issue/pr is a new feature or functionality.
+  - **T-enhancement** - this issue/pr covers improvement of existing functionality.
```
Comment (q9f) [inline docs/docs/repo/labels.md:18]: feature and enhancement is kind of the same

**Review by q9f — APPROVED** [review]

File: docs/docs/repo/labels.md (hunk 3 above)
Comment (dimartiro) [reply docs/docs/repo/labels.md:18]: I understand a feature as something new but an enhancement as an "improvement" of something that is already solved in our code. @P1sar was that your intention?

Comment (P1sar) [reply docs/docs/repo/labels.md:18]: Yeah, my meaning exactly as what Diego means

**Comment by q9f** [comment]
Can you [sign](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) your commits?

**Pull request merged**